python/lsst/meas/astrom/pessimisticPatternMatcherUtils.cc

```diff
@@ -0,0 +1,46 @@
+// -*- LSST-C++ -*-
+
+/*
+ * This file is part of meas_astrom.
+ *
+ * Developed for the LSST Data Management System.
+ * This product includes software developed by the LSST Project
+ * (https://www.lsst.org).
+ * See the COPYRIGHT file at the top-level directory of this distribution
+ * for details of code ownership.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#include "pybind11/pybind11.h"
+#include "pybind11/stl.h"
+#include "numpy/arrayobject.h"
+#include "ndarray/pybind11.h"
+
+#include "lsst/meas/astrom/pessimisticPatternMatcherUtils.h"
+
+namespace py = pybind11;
+
+namespace lsst {
+namespace meas {
+namespace astrom {
+
+PYBIND11_MODULE(pessimisticPatternMatcherUtils, mod) {
+    mod.def("check_spoke", &check_spoke,
+            "Test the opening angle between the spokes of our source pattern against the reference.");
```

Comment:
Huh, I wasn't aware that you could add a python docstring like this. We don't mention it in the dev guide, but we probably should!